src/components/KonnectorSync.jsx

```diff
@@ -1,15 +1,26 @@
+import styles from '../styles/KonnectorSync'
+
 import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 import DescriptionContent from './DescriptionContent'
 
-export const KonnectorSync = ({ t, f, date }) => {
+export const KonnectorSync = ({ t, f, date, error, editing, submitting, onForceConnection }) => {
   return (
     <div>
       { date && <DescriptionContent
         title={t('account.message.synced.title')}
         messages={[t('account.message.synced.last_sync', { date: f(date, t('account.message.synced.date_format')) })]}
       /> }
+      { !editing && !error && <div className={styles['account-forceConnection']}>
+        <button
+          className={submitting ? styles['submitting'] : ''}
```

Comment:
Not really, no. If the button had the class `.coz-btn` there would have been a `::after` with the good size and placed but that's it. It's the modifier class like `.coz-btn--regular`or `.coz-btn--secondary` that sets the right spinner with the right color because of the changing background-color obviously.

In that case, with no class `.coz-btn`, there will be nothing visually to show the busy aspect.